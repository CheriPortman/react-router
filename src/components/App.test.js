import React from 'react';
import { shallow } from 'enzyme';
import App from './App';

describe('App componenet', () => {
  it('renders the app', () => {
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  });
});
