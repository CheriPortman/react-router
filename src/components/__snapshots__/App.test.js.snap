// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App componenet renders the app 1`] = `
<BrowserRouter>
  <Header />
  <Switch>
    <Route
      component={[Function]}
      path="/red"
    />
    <Route
      component={[Function]}
      path="/blue"
    />
    <Route
      componenet={[Function]}
      path="/yellow"
    />
  </Switch>
</BrowserRouter>
`;
